.github/workflows/build.yml

```diff
@@ -62,12 +62,21 @@ jobs:
           cp configs/config.${{ matrix.vendor }}_${{ matrix.model }} .config
           make olddefconfig
           make
+      - name: Create patched binary with pre-enabled serial output
+        run: |
+          cp build/coreboot.rom build/coreboot_serial_enabled.rom
+          build/util/cbfstool/cbfstool build/coreboot_serial_enabled.rom write -r SMMSTORE -f ci/smmstore-serial-enabled
+
+          if [ -e "ci/${{ matrix.vendor }}_${{ matrix.model }}_pretrained" ]; then
+            cp build/coreboot_serial_enabled.rom build/coreboot_serial_enabled_pretrained.rom
+            build/util/cbfstool/cbfstool build/coreboot_serial_enabled_pretrained.rom write -r RW_MRC_CACHE -f ci/${{ matrix.vendor }}_${{ matrix.model }}_pretrained
+          fi
```

Comment:
Why do we need a pretrained binary? This is not a stable solution.

Reply:
@miczyg1 We need to enable serial port but can do this only by manually entering setup and enabling it. This was workaround suggested by @macpijan : https://jira-3mdeb.atlassian.net/browse/DSH-762?focusedCommentId=133640

Reply:
I suggested just enabling serial, not including pretraineded region.

Reply:
Removed: https://github.com/Dasharo/coreboot/pull/432/commits/bf4b3d93192b5778c9a0d6ca75b7a5edf4cb8228#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L70-L73